Lesson23/Dilek/script.js

```diff
@@ -0,0 +1,123 @@
+import { ACCESS_KEY } from './secrets.js';
```

Comment:
For now, you cannot use imports like this due to the structure of the project we have. The workaround is to add script and secrets files to HTML as separate inputs that is done already:
```
<script src="./secrets.js"></script>
<script src="./script.js"></script> 
```

Reply:
Remove 'import { ACCESS_KEY } from './secrets.js';'otherwise code does not work